scripts/ga_airtable_transfer.js

```diff
@@ -0,0 +1,149 @@
+const Airtable = require('airtable')
+const Sheet = require('google-spreadsheet')
+const path = require('path')
+const env = require('node-env-file')
+
+switch (process.env.NODE_ENV) {
+  case 'test':
+    env(path.resolve(__dirname, '../.env.test'))
+    break
+  case 'production':
+  case 'staging':
+    break
+  default:
+    env(path.resolve(__dirname, '../.env'))
+}
+
+console.log('Airtable <> GS start time: ' + new Date().toISOString())
+console.log('Authenticating with Google Sheets and Airtable...')
+
+let rowCount, creds
+
+const base = new Airtable({
+  apiKey: process.env.AIRTABLE_KEY
+}).base(process.env.AIRTABLE_BASE)
+
+const doc = new Sheet(process.env.GOOGLE_SHEET_ID)
+
+if (process.env.NODE_ENV === 'development') {
+  creds = require('../../sheetsjwt.json')
+} else {
+  creds = {
+    client_email: process.env.GOOGLE_SHEET_EMAIL,
+    private_key: process.env.GOOGLE_SHEET_KEY
+  }
+}
+
+function authenticateGoogle () {
+  return new Promise((resolve, reject) => {
+    doc.useServiceAccountAuth(creds, (err, res) => {
+      err ? reject(err) : resolve(res)
+    })
+  })
+}
+
+function docInfo () {
+  return new Promise((resolve, reject) => {
+    doc.getInfo((err, res) => {
+      err ? reject(err) : resolve(res.worksheets[1].rowCount)
+    })
+  })
+}
+
+async function copyDataTask (offset) {
+  try {
+    const getRows = () =>
+      new Promise((resolve, reject) => {
+        doc.getRows(2, { limit: 5, offset: offset }, (err, rows) => {
+          err ? reject(err) : resolve(rows)
+        })
+      })
+    const rows = await getRows()
+    let promises = rows.map((row) => airtableFetch(row))
+    await Promise.all(promises)
+  } catch (err) {
+    console.log(err)
+  }
+}
+
+async function airtableFetch (row) {
+  try {
+    const getRecord = () =>
```

Comment:
Might help clean things up to move these functions outside `airtableFetch` seeing as they don't depend on anything in the function scope like `row`.